common/src/stack/ansible/Makefile

```diff
@@ -0,0 +1,22 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+
+ROLLROOT = ../../../..
+include $(STACKBUILD)/etc/CCRules.mk
+
+build:
+
+install::
+	# Install the Ansible config
+	mkdir -p $(ROOT)/etc/ansible/
+	$(INSTALL) -m0644 etc/ansible.cfg $(ROOT)/etc/ansible/ansible.cfg
+
+	# Install the inventory plugin
+	mkdir -p $(ROOT)/opt/stack/ansible/plugins/inventory
+	$(INSTALL) -m0644 plugins/inventory/stacki.py $(ROOT)/opt/stack/ansible/plugins/inventory/
+
+clean::
```

Comment:
Is this target needed here if nothing is being added to it? Same question with the empty build target. Check throughout.

If the empty targets are needed because makefile magic, please add comments indicating they are needed to the makefile for future travelers.

Reply:
No. I copied other Makefile magic. You don't like it, rewrite all the makefiles.